Update system_info.sh

MirDur opened the pull request:
Skript obsahuje příkazy pro zjištění některých informací o operačním systému Linux.

MirDur committed:
Update system_info.sh

Skript obsahuje příkazy pro zjištění některých informací o operačním systému Linux.

MirDur committed:
Merge branch 'main' into main

MirDur committed:
HW#3

Budete přidávat další funkcionalitu do svého Linux CLI.
Do libovolného adresáře vytvořte složku a uvnitř jeden soubor.
Do souboru zapište řetězec (ne manuálně, ale pomocí nějakého příkazu a bude to přímo součástí všeho CLI).
Vytvořte soft a Hard Link pro daný soubor do /tmp.
Dále script pokračuje vypsáním všech balíčku, které máte nainstalované.
Vyhledá docker balíček.
Na závěr vypíše všechny balíčky, které mohou být upgradovany, ale samotný upgrade neproběhne.
Pokračujte ve svém forku a hotový script pošlete jako Pull Request na review.

Mart-Kuc commented:
Vypada to pekne. Prikazy jsou spravne zvolene. Akorat, proc je pred kazdym prikazem dolar `$`

MirDur commented:
> HW#3

Aby bylo zřetelné, že jsou to příkazy do Linux CLI. 
Opravím to.. :-) A děkuju za review.

MirDur committed:
Update HW#3

Odstranil jsem symboly dolaru na začátku řádků, které se mají provádět. 
A pár dalších menších úprav.

MirDur committed:
Update system_info.sh

MirDur commented:
Několik drobných změn v Bash skriptu <system_info.sh>.
